models/clm/shell_scripts/cesm2_2/datm.streams.txt.CPLHISTForcing.Solar_complete

```diff
@@ -0,0 +1,76 @@
+<streamstemplate>
```

Comment:
to be honest, i'm trusting you on the forcing files and scripts and nml changes here.